transport/grpc/options.go

```diff
@@ -42,10 +42,12 @@ const (
 	// defensive programming
 	// these are copied from grpc-go but we set them explicitly here
 	// in case these change in grpc-go so that yarpc stays consistent
-	defaultServerMaxRecvMsgSize = 1024 * 1024 * 4
 	defaultServerMaxSendMsgSize = math.MaxInt32
-	defaultClientMaxRecvMsgSize = 1024 * 1024 * 4
 	defaultClientMaxSendMsgSize = math.MaxInt32
+	// Overriding default server and client maximum request and response
+	// receive sizes to 64MB.
+	defaultServerMaxRecvMsgSize = 1024 * 1024 * 64
```

Comment:
If server/client has a max recv limit increased shouldn't the other side also be able to send the increased data limit or else sender will have the failure for the limit breach? should be increase all the limits?